packages/@aws-cdk-testing/framework-integ/test/aws-logs/test/integ.log-resource-policy-any-principal.ts

```diff
@@ -0,0 +1,22 @@
+import { App, Stack, StackProps } from 'aws-cdk-lib';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import { LogGroup } from 'aws-cdk-lib/aws-logs';
+import * as iam from 'aws-cdk-lib/aws-iam';
+
+class LogGroupIntegStack extends Stack {
+  constructor(scope: App, id: string, props?: StackProps) {
+    super(scope, id, props);
+    const logGroup = new LogGroup(this, 'LogGroup');
+    logGroup.addToResourcePolicy(new iam.PolicyStatement({
+      effect: iam.Effect.ALLOW,
+      resources: ['*'],
```

Comment:
since this int test may be used as an example, maybe this should be `[logGroup.logGroupArn]` ?  The way it's written now, it results in a policy that says any anonymous principal can write to any log group in the account.

Reply:
One more thought... is `addToResourcePolicy` misleading? It's a method on `LogGroup`, but it actually creates a `AWS::Logs::ResourcePolicy` which is not tied to an `LogGroup` at all, but at the account level. I almost think resources for this method should be set to `logGroup.logGroupArn` and there should be another mechanism to create a resource policy with parameterized resources.

Reply:
I have changed this to the suggested value. I'd recommend that if there are larger concerns about the `addToResourcePolicy` API that they be addressed in a separate issue to make sure that they get the needed attention rather than on a PR targeted at a tangentially related bug fix.